numalogic/registry/artifact.py

```diff
@@ -129,6 +128,14 @@ def construct_key(skeys: KEYS, dkeys: KEYS) -> str:
         return "::".join([_static_key, _dynamic_key])
 
 
+def _apply_jitter(ts: int, jitter_secs: int, jitter_steps_min: int):
+    return random.randrange(
+        min(ts - jitter_secs, ts),
```

Comment:
Is this `min()` redundant?

`min(ts - jitter_secs, ts)` always equals to `ts - jitter_secs` unless jitter_secs is negative or the minus operation underflows.

Reply:
aaahhh....This should have been `max` and not `min`.